manifests/init.pp

```diff
@@ -6,6 +6,6 @@
 class spectacle {
   package { 'spectacle':
     provider => 'compressed_app',
-    source   => 'https://s3.amazonaws.com/spectacle/downloads/Spectacle+0.8.2.zip'
+    source   => 'https://s3.amazonaws.com/spectacle/downloads/Spectacle+${version}.zip'
```

Comment:
Maybe I'm wrong but to this work we need to receive `version` as argument. Is it working this way?